FSharpBuild.Directory.Build.props

```diff
@@ -42,7 +42,10 @@
     <SignAssembly>true</SignAssembly>
     <AssemblyOriginatorKeyFile>$(FSharpSourcesRoot)\fsharp\msft.pubkey</AssemblyOriginatorKeyFile>
     <StrongNames>true</StrongNames>
-    <DelaySign>true</DelaySign>
+    <DelaySign>false</DelaySign>
+    <PublicSign>true</PublicSign>
+    <GenerateSatelliteAssembliesForCore>true</GenerateSatelliteAssembliesForCore>
```

Comment:
Why is this necessary?

Reply:
I mean the GenerateSatelliteAssembliesForCore not the public sign thingy.
Why was that necessary and similarly the NoCompilerStandardLib?

Reply:
The `GenerateSatelliteAssemblies` target does not support public sign.